packages/citation-js-utils/src/index.ts

```diff
@@ -171,12 +164,46 @@ export function firstNonDoiUrl(str?: string, doi?: string) {
   return matches.map((match) => match[0]).find((match) => !doi || !match.includes(doi));
 }
 
-export async function getCitations(bibtex: string): Promise<CitationRenderer> {
```

Comment:
Changes look good, I will test again today!

@agoose77  -- can we get a shim on this that wraps that so this turns into a patch rather than a major release?